client_cli/pytests/common/consts.py

```diff
@@ -0,0 +1,45 @@
+"""
+This module contains constant values and enums used throughout the application.
```

Comment:
```suggestion
Constant values and enumerations used throughout the testing framework. 
```
If Rust is auggestive of a style PEP8 outright dictates style.